integration/test.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+set -ex
+
+_DIR="${BASH_SOURCE%/*}"
+if [[ ! -d "$_DIR" ]]; then _DIR="$PWD"; fi
+
+while ! kubectl wait --for=condition=available deploy/capi-microk8s-bootstrap-controller-manager -n capi-microk8s-bootstrap-system; do
+    echo "Waiting for bootstrap controller to come up"
+    sleep 3
+done
+while ! kubectl wait --for=condition=available deploy/capi-microk8s-control-plane-controller-manager -n capi-microk8s-control-plane-system; do
+    echo "Waiting for control-plane controller to come up"
+    sleep 3
+done
+
+echo "Deploy a test cluster"
+
+# generate cluster
+export AWS_REGION=us-east-1
+export AWS_SSH_KEY_NAME=capi
+export CONTROL_PLANE_MACHINE_COUNT=3
+export WORKER_MACHINE_COUNT=3
+export AWS_CREATE_BASTION=false
+export AWS_PUBLIC_IP=false
+export AWS_CONTROL_PLANE_MACHINE_FLAVOR=t3.large
+export AWS_NODE_MACHINE_FLAVOR=t3.large
+clusterctl generate cluster test-ci-cluster --from "${_DIR}/../templates/cluster-template-aws.yaml" --kubernetes-version 1.25.0 > cluster.yaml
```

Comment:
This make the test infrastructure (AWS) specific. Could we pull this assumption out of the test?